contracts/LockedNORI.sol

```diff
@@ -303,41 +312,57 @@ contract LockedNORI is
   }
 
   /**
-   * @dev revokeUnvestedTokens: Truncates a vesting grant.
+   * @notice batchRevokeUnvestedTokenAmounts: Truncates a batch of vesting grants of amounts in a single go
+   *
+   * @dev
+   * - To revoke all remaining revokable/unvested tokens in a batch, set the amount to 0 in the `amounts` array.
+   * - To revoke tokens at the current block timestamp, set atTimes to 0 in the `amounts` array.
    *
-   * Transfers any unvested tokens in `from`'s grant to `to`
+   * Transfers any unvested tokens in `fromAccounts`'s grant to `to`
    * and reduces the total grant size.
    *
    * No change is made to balances that have vested but not yet been claimed
    * whether locked or not.
-   */
-  function revokeUnvestedTokens(
-    address from,
-    address to,
-    uint256 atTime
-  ) external whenNotPaused onlyRole(TOKEN_GRANTER_ROLE) {
-    _revokeUnvestedTokens(from, to, atTime, 0);
-  }
-
-  /**
-   * @dev revokeUnvestedTokenAmount: Truncates a vesting grant.
    *
-   * Transfers any unvested tokens in `from`'s grant to `to`
-   * and reduces the total grant size.
+   * For revocation by amount regardless of time (must by <= unvested amount):
+   *    *atTime* should be pase.  Amount must be present.
```

Comment:
This comment doesn't make sense ...

I think it's most straightforward to say amount is zero for a revoke by amount and amount is zero for a revoke by time.

Reply:
Hmm. I don't remember making this change even. Will fix

Reply:
Thanks! Looks like merge problems!